pom.xml

```diff
@@ -28,7 +28,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.609.1</version>
+        <version>2.19</version>
```

Comment:
Why do you need this?
I don't see any reason for it. As I've said in previous PRs; When we bump anything here I want to see a more rigorous cleanup for example moving to more recent workflow.version and cleaning up all `@Inject` etc. And that should go in a separate PR.